src/EnergyPlus/SetPointManager.hh

```diff
@@ -269,6 +167,8 @@ namespace SetPointManager {
         SPBase() : CtrlTypeMode(CtrlVarType::Invalid)
         {
         }
+
+        Real64 calcSetPointLinInt(Real64 LowVal, Real64 HighVal, Real64 RefVal, Real64 SetptAtLowVal, Real64 SetptAtHighVal);
```

Comment:
Moved `calcSetPointLinInt` to the `SPBase` base class. I think this is what @Myoldmopar was suggesting.